.ci/setup_env_centos.sh

```diff
@@ -22,7 +22,7 @@ fi
 echo "skip_missing_names_on_install=0" | sudo tee -a /etc/yum.conf
 
 # Check EPEL repository is enabled on CentOS
-if [ -z $(yum repolist | grep "Extra Packages") ]; then
+if [ -z "$(yum repolist | grep "Extra Packages")" ]; then
```

Comment:
Nit: I find double sets of double quotes confusing so since the grep expression doesn't contain variables, you could make it:

```suggestion
if [ -z "$(yum repolist | grep 'Extra Packages')" ]; then
```

Reply:
@jodh-intel yeah, feels like double negative sometimes. I accepted your suggestion on the version I just pushed.